src/components/badges/BadgeWrapper.tsx

```diff
@@ -31,7 +33,8 @@ export default function ({
     <div
       className={classNamesToString(
         badgeWrapper(minted),
-        'last:odd:col-span-full'
+        'each-new-line:last:col-span-full',
```

Comment:
I hope this is understandable, each new line (badges number 1, 4, 7...) will stretch to full line